internal/cli/alerts/settings_update.go

```diff
@@ -38,6 +38,8 @@ func (opts *ConfigUpdateOpts) initStore() error {
 	return err
 }
 
+var updateTemplate = "Updating alert configuration {{.ID}}.\n"
```

Comment:
similar to create this can be just 
```suggestion
var updateTemplate = "Alert configuration '{{.ID}}' updated.\n"
```

Reply:
you put single quote for `'{{.ID}}'`, is something I should use for each IDs?

Reply:
Just for the create/update/ack ones to separate the ID part from the text, we do the same with delete